src/Engine/Manager/PhysicsManager.hpp

```diff
@@ -35,6 +35,13 @@ class PhysicsManager : public SuperManager {
          */
         void Update(World& world, float deltaTime);
 
+        /// Update transforms of entities according to positions of physics
+        /// components.
+        /**
+         * @param world The world in which components reside? Wat?
```

Comment:
Wat?

Reply:
@NinjaDanz3r  I'm not sure about the purpose of world here and followed what has been done before. Do you know why world must be passed? This also happens in PhysicsManager::Update.